homeassistant/components/hue/bridge.py

```diff
@@ -105,6 +124,8 @@ async def async_setup(self, tries=0):
             3 if self.api.config.modelid == "BSB001" else 10
         )
 
+        self.config_entry.add_update_listener(_update_listener)
```

Comment:
We need to unlisten when we unload config entry

Reply:
PS: Don't search for `config_entry.add_update_listener` in our code base for this case... 😓

Reply:
Our docs set a bad example:
https://developers.home-assistant.io/docs/config_entries_options_flow_handler#signal-updates

Reply:
I just looked at our code, we actually use weak refs, so unsubscribing is not _perse_ necessary, as it will be automatically unsubscribed when last reference is purged from memory. Still we can do better.

Reply:
Updated docs https://github.com/home-assistant/developers.home-assistant/commit/0f65f56f701a2ec2d286a3ac23c0a4b9ea05a84c